docfiles/docs.js

```diff
@@ -47,8 +54,21 @@ function searchSubmit(form) {
 }
 
 function setupSidebar() {
+    let togglesidebar = document.getElementById("togglesidebar");
+    togglesidebar.onkeydown = handleEnterKey;
```

Comment:
The docs will never migrate to react (or very very unlikely). We already use JQuery extensively in this part of the house when rendering snippets.